frontend_tests/node_tests/recent_topics.js

```diff
@@ -707,17 +731,6 @@ test("basic assertions", ({override_rewire, mock_template}) => {
         "4:topic-10,1:topic-7,1:topic-6,1:topic-5,1:topic-4,1:topic-3,1:topic-2,1:topic-1",
     );
 
-    rt_data.process_message({
-        type: "private",
-    });
-
-    // Private msgs are not processed.
-    assert.equal(all_topics.size, 8);
-    assert.equal(
-        Array.from(all_topics.keys()).toString(),
-        "4:topic-10,1:topic-7,1:topic-6,1:topic-5,1:topic-4,1:topic-3,1:topic-2,1:topic-1",
-    );
```

Comment:
We should replace this test with something verifying the processing of private messages :)